### ISSUE TITLE
Automated docs would be helpful

### ISSUE THREAD

**Issue by erikjung** [cloudfour/drizzle-builder#7]
Related to #6

**Comment by lyzadanger** [comment]
FYI, I tried `doxdox` documentation rendering. Despite my relatively robust comments, the output didn't seem too...useful. You're more than welcome to see if you can get it to be a bit nicer!

**Comment by neogeek** [comment]
Hey there, I'm the creator of `doxdox` and I saw you were looking into using it for this project. I'm sorry the output isn't working for you - I'm actually in the process of writing a new version (with more in-depth documentation) and would love a chance to work with you to improve the output.

**Comment by erikjung** [comment]
Hello @neogeek,

Great work on `doxdox`. We've only used it on one other project so far, but it's been an enjoyable tool for producing standardized README docs (using the `--layout markdown` option).

Regarding the output, I believe the only issue we've struggled with is how to use the Markdown layout (outputting to a single file), but with more control over the order things are displayed in. 

For a very simplified example, we might have a structure like this:

```
src
├── helpers.js
└── main.js
```

```sh
doxdox src --output docs/README.md --layout markdown
```

In the resulting file, the helper functions would come first, while we might prefer them to come last. Aside from renaming files, it would be helpful to be able to specify multiple path arguments (which would be read in series), for example:

```sh
doxdox src/main.js src/helpers.js --output docs/README.md --layout markdown
```

**Comment by neogeek** [comment]
@erikjung: That is a very good point, and something I hadn't considered before. I will look into seeing how easy it would be to add multiple input files through the current CLI. Do you think it would also be useful to create a config file for projects with more than a couple of files?

Something like this:

**.doxdoxrc**

```json
{
    "input": [
        "src/main.js",
        "src/helpers.js"
    ]
}
```

**Comment by erikjung** [comment]
@neogeek 

> Do you think it would also be useful to create a config file for projects with more than a couple of files?

I can't think of a reason why not :) 

I assume the `input` array would dictate the order of concatenation when the docs are built?

**Comment by neogeek** [comment]
@erikjung 

> I assume the input array would dictate the order of concatenation when the docs are built?

Yep. Going to look into adding the CLI feature this weekend and then porting it over to the rebuild.

The config file and glob based pattern matching might need to wait until the next major release (currently in development).

**Comment by neogeek** [comment]
@erikjung

I have released a new version of doxdox (`v0.1.13`) that supports multiple files via both the command line script and API.

Let me know if you run into any issues.

**Comment by erikjung** [comment]
Thanks @neogeek,

I tried out the multiple path arguments, and it works just as expected 👍
